test/distributed_exec.jl

```diff
@@ -661,7 +661,20 @@ catch ex
     @test typeof(ex.exceptions[1].ex) == ErrorException
     errors = map(x->x.ex.msg, ex.exceptions)
     @test collect(1:5) == sort(map(x->parse(Int, x), errors))
-end
+    # test start, next, and done
+    for (i, i_ex) in enumerate(ex)
+        @test i == parse(Int, i_ex.ex.msg)
+    end
+    # test showerror
+    err_strs = split(sprint(showerror, ex), "\n")
```

Comment:
Hmm, you can split it over 3 lines then. Have `err_str`, `err_one_str` (without the `split`) and then have the test